<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsEdit</class>
 <widget class="QDialog" name="SettingsEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/icons.qrc">
    <normaloff>:/icons/48x48/settings.png</normaloff>:/icons/48x48/settings.png</iconset>
  </property>
  <layout class="QGridLayout" name="layout">
   <item row="0" column="0">
    <widget class="QListWidget" name="setttingsListWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="gridSize">
      <size>
       <width>90</width>
       <height>80</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="../../../data/icons.qrc">
        <normaloff>:/icons/48x48/settings.png</normaloff>:/icons/48x48/settings.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channels</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="../../../data/icons.qrc">
        <normaloff>:/icons/48x48/video.png</normaloff>:/icons/48x48/video.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Interface</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="../../../data/icons.qrc">
        <normaloff>:/icons/48x48/interface.png</normaloff>:/icons/48x48/interface.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Playback</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="../../../data/icons.qrc">
        <normaloff>:/icons/48x48/playback.png</normaloff>:/icons/48x48/playback.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Recorder</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="../../../data/icons.qrc">
        <normaloff>:/icons/48x48/recorder.png</normaloff>:/icons/48x48/recorder.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shortcuts</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="../../../data/icons.qrc">
        <normaloff>:/icons/48x48/shortcuts.png</normaloff>:/icons/48x48/shortcuts.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="settingsWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageMain">
      <layout class="QVBoxLayout" name="layoutMain">
       <item>
        <widget class="QGroupBox" name="tanoBox">
         <property name="title">
          <string>Tano</string>
         </property>
         <layout class="QGridLayout" name="layoutTanoBox">
          <item row="0" column="0">
           <widget class="QLabel" name="labelVersion">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="wizardCheck">
            <property name="text">
             <string>Run First run wizard after restart</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="3">
           <widget class="QLabel" name="labelLogo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../data/logo.qrc">:/logo/64x64/logo.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="tanoBoxSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="langBox">
         <property name="title">
          <string>Language</string>
         </property>
         <layout class="QVBoxLayout" name="layoutLangBox">
          <item>
           <widget class="QRadioButton" name="defaultLanguageRadio">
            <property name="text">
             <string>System default language</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="customLanguageRadio">
            <property name="text">
             <string>Custom language</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="languageComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="sessionBox">
         <property name="title">
          <string>Session store</string>
         </property>
         <layout class="QVBoxLayout" name="layoutSessionBox">
          <item>
           <widget class="QCheckBox" name="sessionVolumeCheck">
            <property name="text">
             <string>Remember volume</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sessionAutoplayCheck">
            <property name="text">
             <string>Autoplay last channel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="generalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>156</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageChannels">
      <layout class="QVBoxLayout" name="layoutChannels">
       <item>
        <widget class="SettingsPlaylist" name="widgetSelectPlaylist" native="true"/>
       </item>
       <item>
        <spacer name="channelsSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>273</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageInterface">
      <layout class="QVBoxLayout" name="layoutInterface">
       <item>
        <widget class="QGroupBox" name="mainGuiBox">
         <property name="title">
          <string>General Interface Settings</string>
         </property>
         <layout class="QGridLayout" name="layoutMainGuiBox">
          <item row="3" column="0">
           <widget class="QLabel" name="labelToolbarStyle">
            <property name="text">
             <string>Toolbar style:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QComboBox" name="toolbarLookComboBox">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>Icon only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Text only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Text beside icon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Text under icon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>System default</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="osdCheck">
            <property name="text">
             <string>Enable OSD in Fullscreen mode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelMouseWheel">
            <property name="text">
             <string>Mouse wheel function:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="radioWheelChannel">
            <property name="text">
             <string>Change channel</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QRadioButton" name="radioWheelVolume">
            <property name="text">
             <string>Change volume</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QCheckBox" name="trayCheck">
            <property name="text">
             <string>Minimise to tray</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="startupGuiBox">
         <property name="title">
          <string>Default options for startup</string>
         </property>
         <layout class="QVBoxLayout" name="layoutStartupGuiBox">
          <item>
           <widget class="QCheckBox" name="checkSplash">
            <property name="text">
             <string>Show splash screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkLite">
            <property name="text">
             <string>Simple mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkTop">
            <property name="text">
             <string>Always on top</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkControls">
            <property name="text">
             <string>Controls enabled</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkInfo">
            <property name="text">
             <string>Info panel enabled</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="guiSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>209</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePlayback">
      <layout class="QVBoxLayout" name="layoutPlayback">
       <item>
        <widget class="QGroupBox" name="vlcqtBox">
         <property name="title">
          <string>VLC-Qt Backend</string>
         </property>
         <layout class="QGridLayout" name="layoutVlcqtBox">
          <item row="0" column="0">
           <widget class="QLabel" name="labelVlcqt">
            <property name="text">
             <string>Curently using VLC-Qt Backend</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelVlcqtVersion">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="vlcGlobalCheck">
            <property name="text">
             <string>Use global VLC settings (requires restart)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="5">
           <widget class="QLabel" name="labelVlcqtIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../data/icons.qrc">:/icons/48x48/vlc-qt.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelVlcVersion">
            <property name="text">
             <string>Using Videolan VLC version:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="playbackBox">
         <property name="title">
          <string>Playback</string>
         </property>
         <layout class="QGridLayout" name="layoutPlaybackBox">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkVideoSettings">
            <property name="text">
             <string>Remember Video settings between media</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelLSub">
            <property name="text">
             <string>Default subtitle language:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelLAudio">
            <property name="text">
             <string>Default audio language:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboSub">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">English</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Slovenščina / Slovenian </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Deutsch / German</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Hrvatski / Croatian</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Other</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboAudio">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">English</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Slovenščina / Slovenian </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Deutsch / German</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Hrvatski / Croatian</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Other</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="playbackSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>320</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRecorder">
      <layout class="QVBoxLayout" name="layoutRecorder">
       <item>
        <widget class="QCheckBox" name="enableRecorderCheck">
         <property name="text">
          <string>Enable Recorder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="defaultDirBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Default recording directory</string>
         </property>
         <layout class="QHBoxLayout" name="layoutDefaultDirBox">
          <item>
           <widget class="QLineEdit" name="recorderDirectoryLineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonBrowseRecorder">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonResetRecorder">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
            <property name="icon">
             <iconset resource="../../../data/icons.qrc">
              <normaloff>:/icons/24x24/refresh.png</normaloff>:/icons/24x24/refresh.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="recorderSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageShortcuts">
      <layout class="QVBoxLayout" name="layoutShortcuts">
       <item>
        <widget class="SettingsShortcuts" name="shortcutsWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingsPlaylist</class>
   <extends>QWidget</extends>
   <header>ui/settings/SettingsPlaylist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsShortcuts</class>
   <extends>QWidget</extends>
   <header>ui/settings/SettingsShortcuts.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/icons.qrc"/>
  <include location="../../../data/logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>enableRecorderCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>defaultDirBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setttingsListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>settingsWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customLanguageRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>languageComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
